src/utils/components/PolicyForm/PolicyWizard/ReviewStep.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { FC } from 'react';
+import { NodeModelGroupVersionKind } from 'src/console-models/NodeModel';
+
+import { IoK8sApiCoreV1Node } from '@kubevirt-ui/kubevirt-api/kubernetes/models';
+import { useK8sWatchResource } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Alert,
+  AlertVariant,
+  Content,
+  Form,
+  Level,
+  LevelItem,
+  Stack,
+  StackItem,
+  Title,
+} from '@patternfly/react-core';
+import {
+  InterfaceType,
+  NodeNetworkConfigurationInterface,
+  V1NodeNetworkConfigurationPolicy,
+} from '@types';
+import { NODE_HOSTNAME_LABEL } from '@utils/constants';
+import { useNMStateTranslation } from '@utils/hooks/useNMStateTranslation';
+import { filterPolicyAppliedNodes } from '@utils/policies/utils';
+
+import InfoBox from './components/InfoBox';
+import InterfaceReview from './components/InterfaceReview';
+
+type ReviewStepProps = {
+  policy: V1NodeNetworkConfigurationPolicy;
+  error?: Error;
+};
+
+const ReviewStep: FC<ReviewStepProps> = ({ policy, error }) => {
+  const { t } = useNMStateTranslation();
+
+  const ethernetInterfaces: NodeNetworkConfigurationInterface[] =
```

Comment:
Consider pulling these out as functions in a util file to increase readability and further separate logic from presentation.

`const ethernetInterfaces: NodeNetworkConfigurationInterface[] = getEthernetInterfaces(policy);`